MainModule/Server/Commands/HeadAdmins.lua

```diff
@@ -137,22 +157,22 @@ return function(Vargs, env)
 
 		UnGameBan = {
 			Prefix = Settings.Prefix;
-			Commands = {"unglobalban", "unpermban", "unpermanentban", "unpban", "ungameban", "ungban"};
```

Comment:
unglobalban is a useful alias, in fact, it's the first alias in the string, why remove this?